run_array_analyzer.py

```diff
@@ -42,15 +50,16 @@ def parse_args():
     return parser.parse_args()
 
 
-def run_workflow(input_dir, output_dir, method, debug=False):
+def run_workflow(input_dir, output_dir, format, array_detection, debug=False):
```

Comment:
I think i prefer `method` over `array_detection`.  `method` could apply to any format array.  For example, we would calculate mean instead of median (or something else) for `well` format

Reply:
How about calling it `workflow`? https://github.com/czbiohub/serology-COVID19/blob/aff4081e1e94d5d89de6c970230a739c03d3461d/run_array_analyzer.py#L64-L90

Reply:
> How about calling it `workflow`?

I think that's fine.  These flags can be adjusted later as more features are added.